42_grml

```diff
@@ -116,6 +116,7 @@ ${grub_prep}
         export iso_path
         kernelopts=" $CUSTOM_BOOTOPTIONS $additional_param "
         export kernelopts
+        rmmod tpm
```

Comment:
i think that needs at least a comment why this is here.

Reply:
Good idea, thanks, done!